include/staff/staff.inc.php

```diff
@@ -337,6 +337,7 @@ class="error">*</span></div>
             if (!isset($permissions[$g]))
                 $permissions[$g] = array();
             $permissions[$g][$k] = $P;
+            asort($permissions[$g]);
```

Comment:
I don't think you mean to put this here? It's inside the `foreach` for `$k`, so you'll be sorting it for each key added to `$permissions[$g]`